Nomster/CreateSuggestionViewController.swift

```diff
@@ -86,11 +86,11 @@ class CreateSuggestionViewController : UITableViewController {
         }
         
         // Dummy values
-        let image = "nomster-logo-large"
+        let image = ""
```

Comment:
@tscholze: We need to set an URL here. I've set it to an empty string to let the app still work with both our changes. That means we need to upload the image somewhere first.